tools/operator/launcher/k8s/network.go

```diff
@@ -99,7 +99,7 @@ func (k8s K8s) launchObject(nsConfig networkspec.Config) ([]LaunchConfig, error)
 				container = corev1.Container{
 					Name:      "couchdb",
 					Resources: k8s.resources(nsConfig.K8s.Resources.Couchdb),
-					Image:     fmt.Sprintf("hyperledger/fabric-couchdb"),
+					Image:     fmt.Sprintf("couchdb:2.3"),
```

Comment:
actually we need not have fmt.Sprintf . May be can we move couchdb image to dockerImages list if given (later).

Reply:
Fixed, and agreed, on the dockerimages